### PR TITLE
Update js-sdk to correct API usage error

### PR DESCRIPTION
**Pull request by LeeAlephium**
Resolves https://github.com/alephium/desktop-wallet/issues/259

### CODE REVIEW

**Comment by LeeAlephium** [comment]
Very weird "dmg-license not accessible from dmg-builder" error has appeared and I've been trying to resolve it. Just recording that I have been poking at it...

**Comment by LeeAlephium** [comment]
We need to be very careful to not destroy the package-lock... Seems electron-builder has updated things and it will break building for now.

I can focus on updating our electron-builder version in another PR (maybe update all our dependencies)?

**Comment by nop33** [comment]
That means that the currently installed wallets (1.2.0) will break once the hardfork (node 1.4.0) is released to mainnet, just like it did with our previous release (1.3.0), correct?

**Comment by LeeAlephium** [comment]
Yep, so it'll be a necessary update

**Comment by nop33** [comment]
> The old wallet is easily detectable via no Accept header, and newer wallets are by seeing Accept header and replying from the appropriate server instance.

This would require changes on our server to read the `Accept` header and accordingly route the request, correct?

File: package.json [hunk 1]
```diff
@@ -38,7 +38,7 @@
     "electron-is-dev": "^2.0.0"
   },
   "devDependencies": {
-    "@alephium/sdk": "0.0.5",
+    "@alephium/sdk": "0.0.11",
```
Comment (nop33) [inline package.json:41]: We probably want `0.0.13-rc.0` here.

**Comment by LeeAlephium** [comment]
This PR is in a weird state because it refers to an older js-sdk version with the fix but using a newer one actually doesn't include the fix. For sanity I'm going to close it and open a new one when js-sdk has the updated node and explorer APIs. I would also like to give it a better branch name like "1.4.0-compat".

**Pull request closed without merging**